src/Compilers/Core/Portable/Compilation/Compilation.cs

```diff
@@ -1397,6 +1397,16 @@ IAssemblySymbol computeContainingAssembly(ISymbol s)
 
         internal abstract void GetDiagnostics(CompilationStage stage, bool includeEarlierStages, DiagnosticBag diagnostics, CancellationToken cancellationToken = default);
 
+        /// <summary>
+        /// Unique metadata assembly references that are considered as used by this compilation.
+        /// For example, if a type declared in a referenced assembly is referenced in source code 
+        /// within this compilation, the reference is considered to be used. Etc.
+        /// The returned set is a subset of references returned by <see cref="References"/> API.
+        /// If compilation contains errors, it is valid to consider all assembly references as used for the purpose
```

Comment:
>If compilation [](start = 12, length = 14)

Nit: `If the compilation`

Reply:
I would suggest replacing these two sentences with "The result is undefined if the compilation contains errors."

---
In reply to: [346559554](https://github.com/dotnet/roslyn/pull/39807#discussion_r346559554) [](ancestors = 346559554)